src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -131,6 +131,22 @@ protected override void ProcessRecord()
                             }
                         }
 
+                        //RootModule can be null, empty string or point to a valid .psm1, or .dll.  Anything else is invalid.
```

Comment:
In testing a fix for cdxml/XAML modules, I see that you get this error message for InvalidModuleManifest which just seems wrong:

https://github.com/PowerShell/PowerShell/blob/309bf603f9aff9009dce7e725d42b2d4f99f58c9/src/System.Management.Automation/resources/Modules.resx#L157

    <value>The module manifest '{0}' could not be processed because it is not a valid Windows PowerShell restricted language file. Remove the elements that are not permitted by the restricted language: {1}</value>

I'd like to change it to something like:

The module manifest '{0}' could not be processed because it is not a valid PowerShell module manifest file.  Remove the elements that are not permitted: {1}

CC @HemantMahawar